pkg/gofr/http/multipart_file_bind.go

```diff
@@ -1,14 +1,30 @@
 package http
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
 	"io"
 	"mime/multipart"
 	"reflect"
 	"strconv"
+	"strings"
 
 	"gofr.dev/pkg/gofr/file"
 )
 
+var (
+	errNilInterface             = errors.New("cannot set value on a nil interface")
+	errUnsupportedInterfaceType = errors.New("unsupported interface value type")
+	errDataLengthExceeds        = errors.New("data length exceeds array capacity")
```

Comment:
I would use past tense for the variable name


```suggestion
	errDataLengthExceeded        = errors.New("data length exceeds array capacity")
```

Reply:
yups sure we can use this